src/infra/marketplace/assets/joomla.json

```diff
@@ -42,6 +42,35 @@
     "cd %installDirectory%; php %installDirectory%/installation/joomla.php install --site-name=baseWebsite --admin-user=%adminUsername% --admin-username=%adminUsername% --admin-password=%adminPassword% --admin-email=%adminMailAddress% --db-type=mysqli --db-host=localhost --db-user=joomlaUser_%installUuid% --db-pass=%installRandomPassword% --db-name=joomlaDb_%installUuid% --db-prefix=joomla_ --db-encryption=0 -n",
     "cp %installDirectory%/htaccess.txt %installDirectory%/.htaccess"
   ],
+  "uninstallCmdSteps": [
+    "os db delete-user -n joomlaDb_%installUuid% -t mysql -u joomlaUser_%installUuid%",
+    "os db delete -n joomlaDb_%installUuid% -t mysql"
+  ],
+  "filesToDelete": [
+    ".htaccess",
+    "LICENSE.txt",
+    "README.txt",
+    "administrator",
+    "api",
+    "cache",
+    "cli",
+    "components",
+    "configuration.php",
+    "htaccess.txt",
+    "images",
+    "includes",
+    "index.php",
+    "language",
+    "layouts",
+    "libraries",
+    "media",
+    "modules",
+    "plugins",
+    "robots.txt",
+    "templates",
+    "tmp",
+    "web.config.txt"
+  ],
```

Comment:
JSON allows for list to be a single line, I wouldn't line break here. You're also mixing directories with files. Why not listing the directories first and then the files as all file managers do? It's more intuitive.

Reply:
Adjusted!